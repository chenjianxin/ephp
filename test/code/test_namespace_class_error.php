<?php
namespace my\name;

class SubNS\MyClass {}
