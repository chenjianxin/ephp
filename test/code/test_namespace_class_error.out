
Parse error: syntax error, unexpected '\' (T_NS_SEPARATOR), expecting '{' in {{CWD}}/test/code/test_namespace_class_error.php on line 4
